### PR TITLE
Functional Tests : BO - Modules - Alerts tab - Disable / Enable mobile

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Functional Tests : BO - Modules - Alerts tab - Disable / Enable mobile
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: & Nightly is :green_circle: : https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/4956734511
| Fixed ticket?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/02_alerts/01_enableDisableMobile.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      dashboardPage.moduleManagerLink,
+    );
+    await moduleManagerPage.closeSfToolBar(page);
+
+    const pageTitle = await moduleManagerPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(moduleManagerPage.pageTitle);
+  });
+
+  it('should go to \'Alerts\' tab', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToModuleManagerPage', baseContext);
```
Comment (nesrineabdmouleh) [inline tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/02_alerts/01_enableDisableMobile.ts]:
```suggestion
    await testContext.addContextItem(this, 'testIdentifier', 'goToAlertsTab', baseContext);
```

Comment (Progi1984) [reply tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/02_alerts/01_enableDisableMobile.ts]: Fixed

**Comment by Progi1984** [comment]
Ping @nesrineabdmouleh & @boubkerbribri

File: tests/UI/pages/BO/modules/moduleManager/index.ts [hunk 2]
```diff
@@ -181,6 +186,15 @@ class ModuleManager extends BOBasePage {
   Methods
    */
 
+  /**
+   * Go to the Alerts Tab
+   * @param page {Page} Browser tab
+   * @return {Promise<boolean>}
```
Comment (nesrineabdmouleh) [inline tests/UI/pages/BO/modules/moduleManager/index.ts]:
```suggestion
   * @return {Promise<void>}
```

Comment (Progi1984) [reply tests/UI/pages/BO/modules/moduleManager/index.ts]: Fixed

**Review by nesrineabdmouleh — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Pull request merged**